stetho/src/main/java/com/facebook/stetho/inspector/network/DecompressionHelper.java

```diff
@@ -20,8 +20,8 @@
 
 // @VisibleForTest
 public class DecompressionHelper {
```

Comment:
Ack'd, though there are other, deeper problems which prevent this.  The API would need a lot of work